Library/Formula/pmd.rb

```diff
@@ -25,6 +25,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}”
```

Comment:
Sorry. What do you recommend I do then?

a) Work around the problem for now by moving this into the `install` function
b) Remove this line entirely and just let CI be broken for now
c) Maintain my own fork until the problem @DomT4 mentioned is fixed

Sorry, I'm new to Homebrew development (didn't even know there was even a Linux version until last week!) so I'm not sure about the protocol here.